src/core/client/framework/helpers/onPymMessageOnce.ts

```diff
@@ -0,0 +1,18 @@
+import { Child, MessageCallback } from "pym.js";
+
+import onPymMessage from "./onPymMessage";
+
+function onPymMessageOnce(
+  pym: Child,
+  messageType: string,
+  callback: MessageCallback
+) {
+  const dispose = onPymMessage(pym, messageType, (raw) => {
+    callback(raw);
+    // Dispose next frame to not disrupt pym handling messages.
+    setTimeout(() => dispose());
```

Comment:
Discussion about moving `setTimeout` directly into `dispose()`

Reply:
I'm against it, because sometimes I want `dispose` to happen right away so to be sure that another event callback is not called. We should investigate that error in sentry why and when it's happening and target that specifically.